.github/workflows/golang_linux.yml

```diff
@@ -27,7 +35,7 @@ jobs:
           - job: record_build_replay_build
             record_src: build
             replay_src: build
-    name: ${{ matrix.app.name }} (${{ matrix.config.job }})
+    name: ${{ matrix.app.name }} (${{ matrix.config.job }})${{ contains(matrix.app, 'enable_ssl') && format(' ssl-{0}', matrix.app.enable_ssl) || '' }}
```

Comment:
I think this condition is not working correctly the name is same for the enabled ssl and also for the disabled one